mlrun/feature_store/steps.py

```diff
@@ -117,7 +173,7 @@ def _map_value(self, feature: str, value):
     def _feature_name(self, feature) -> str:
```

Comment:
functions and methods should be named as verbs, so change it to either get or parse feature name :)